test/cql-pytest/test_filtering.py

```diff
@@ -431,3 +431,14 @@ def test_token_bind_marker_name(cql, test_keyspace):
     with new_test_table(cql, test_keyspace, 'p1 int, p2 int, c1 int, c2 int, r int, PRIMARY KEY ((p1, p2), c1, c2)') as table:
         stmt = cql.prepare(f'SELECT * FROM {table} WHERE token(p1, p2) = ?')
         assert stmt.column_metadata[0].name == 'partition key token'
+
+
+# In the following test we check what error message appears if multi-column relation contains null values. 
+# We also check in which column the incorrect data was entered.   
```

Comment:
The last line in the comment is not (no longer?)  correct.

Also, please add a test for the wrong-number-of-items-in-the-tuple (too many and too few) - you said "I tested it myself", but it's better to have a regression test than testing it once manually.

Reply:
Where should i put this test?

Reply:
maybe the same test_filtering.py where you already put a test? It's yet another case of filtering - it's just an error case (a filtering which isn't allowed because it has the wrong number of items in the tuple),